components/camel-smb/src/main/java/org/apache/camel/component/smb2/Smb2Configuration.java

```diff
@@ -44,18 +51,26 @@ public class Smb2Configuration extends GenericFileConfiguration {
     private String password;
     @UriParam(label = "security", description = "The user domain")
     private String domain;
+    @UriParam(defaultValue = "*.txt", description = "The search pattern used to list the files")
```

Comment:
I dont think we should only include txt files, it should be all files.

Reply:
Makes sense, now includes all files.